src/components/Clock.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+
+      // eslint-disable-next-line
+      console.log(this.state.date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    const { clockName } = this.props;
+    const { date } = this.state;
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {clockName}
+        </strong>
+
+        {' time is '}
```

Comment:
```suggestion
        time is
```
you can write the plain text 😉